framework/src/main/java/org/apache/drill/test/framework/DrillDefaults.java

```diff
@@ -33,7 +33,7 @@
 	/**
 	 * Default line break for logging.
 	 */
-	static final String LINE_BREAK = "----------------------------------------------------------------------------------------------------------------\n";
+	static final String LINE_BREAK = "------------------------------------------------------------------------";
```

Comment:
This is to match standard new line breaker in length. Copied from Maven. Looks cleaner.